src/js/components/pipelines/HoverField.jsx

```diff
@@ -110,15 +117,41 @@ var HoverField = React.createClass({
     this.setState({fieldDef: fieldDef});
   },
 
+  sortValues: function(evt) {
+  // will need to clean up this code
+  var valuesInc = this.state.valuesInc,
+      props = this.props,
+      field = props.def,
+      id = props.dsId,
+      incOrDec = null;
+
+  if (valuesInc == null) {
+    // first click default: increasing
+    valuesInc = true;
+  } else {
+    valuesInc = !valuesInc;
+  }
+  this.setState({ valuesInc : valuesInc });
+
+  incOrDec = valuesInc ? 'inc' : 'dec';
```

Comment:
Let's use `asc` or `desc` instead to conform to typically expected sort order values. (Be sure to update these throughout).